xml/favicons.xml

```diff
@@ -1962,4 +1962,12 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:mitre:caldera:-"/>
   </fingerprint>
 
+  <fingerprint pattern="^531b63a51234bb06c9d77f219eb25553$">
+    <description>phpMyAdmin web interface for MySQL and MariaDB</description>
+    <example>531b63a51234bb06c9d77f219eb25553</example>
+    <param pos="0" name="service.vendor" value="phpmyadmin"/>
```

Comment:
Here and elsewhere for `.vendor` and `.product`: This should be `phpMyAdmin`

Reply:
Good catch!